ga4gh/datamodel/ontologies.py

```diff
@@ -32,7 +33,27 @@ def getId(self, name):
     def getName(self, id_):
         return self._nameIdMap[id_]
 
+    def getGaTermByName(self, name):
```

Comment:
I thought that I inherited the ontologies code as introduced by variant annotations wholesale (that's already in master). I'll check what's going on here.